flake.nix

```diff
@@ -0,0 +1,160 @@
+{
+  inputs = {
+    fenix = {
```

Comment:
@6r1d We need [fenix](https://github.com/nix-community/fenix) documentation in the tutorial at some point. 

@QuentinI do you think we can conduct a half-internal, half-external seminar on `nix`?

Reply:
> We need [fenix](https://github.com/nix-community/fenix) documentation in the tutorial at some point.

fenix can probably be viewed as an implementation detail

> do you think we can conduct a half-internal, half-external seminar on nix?

Sure